website/docs/advanced/docker/_category_.json

```diff
@@ -1,6 +1,6 @@
 {
-  "label": "Docker Images",
-  "position": 7,
+  "label": "Docker builds",
```

Comment:
```suggestion
  "label": "Docker Builds",
```

Reply:
Using lower case on all the other pages so I'm going to leave this alone for now